### ISSUE TITLE
Certain fonts can't be set correctly

### ISSUE THREAD

**Issue by afraidofmusic** [johnfactotum/foliate#589]
**Describe the bug**
Trying to set the font to a font containing multiple spaces results in the wrong font being set.

**To Reproduce**
1. Open a document.
2. Set a font containing multiple spaces, for example "Noto Serif Condensed"

**Expected behavior**
Document is rendered using the intended font.

**Screenshots**

![Screenshot from 2020-09-07 12-20-24](https://user-images.githubusercontent.com/25895258/92372570-ab977780-f0ec-11ea-8950-f46a9a0c6f6c.png)

![Screenshot from 2020-09-07 12-20-18](https://user-images.githubusercontent.com/25895258/92372573-acc8a480-f0ec-11ea-8792-a0cedf65f700.png)

**Version:**
 - Foliate version: 2.4.2
 - OS/Distribution and version: Arch Linux
 - Desktop environment: 3.36.5
 - Installation method: pacman

**Additional context**
The problem seems to be caused by a missing comma. If a comma is placed before the font size in dconf, the correct font is displayed.

**Comment by johnfactotum** [comment]
The format without the comma is actually the correct one, as far as GTK and Pango is concerned.

When Foliate reads the font from the GTK font chooser widget, it's not possible (AFAIK) to get a string that can be used as-is in CSS. Instead you can only get the family name, the weight, the stretch, the size, and so on.

When you choose "Noto Serif Condensed" in the font chooser, you're actually choosing the family `Noto Serif`, and the strech `Condensed`. Problem is — `font-stretch` is not supported by WebKit, so while Foliate knows that the font should be condensed, and sets the CSS accordingly, in the end it has no effect.

Now, when you add the comma, you're setting the _family_ name as `Noto Serif Condensed`. This works because CSS does understand that as a family name. However, GTK/Pango doesn't understand it; it only knows the `Noto Serif` family plus the `Condensed` stretch, but not any family called `Noto Serif Condensed`, and if you look at the font chooser you'd see it's displayed as "None".

In short, if I'm not mistaken, it's both a GTK/Pango bug and an WebKit bug.

**Comment by afraidofmusic** [comment]
Thank you.
